QuantConnect.TDAmeritrade/TDAmeritradeSymbolMapper.cs

```diff
@@ -61,9 +61,24 @@ public Symbol GetLeanSymbol(string brokerageSymbol, SecurityType securityType, s
         /// <exception cref="ArgumentException">Wrong Lean <see cref="Symbol"/></exception>
         public string GetBrokerageWebsocketSymbol(Symbol symbol)
         {
-            _wsSymbolMap.TryAdd(symbol.Value, symbol);
+            var brokerageSymbol = GetBrokerageSymbol(symbol);
 
-            return symbol.ID.Symbol;
+            if(brokerageSymbol.Contains('.'))
+            {
+                brokerageSymbol.Replace('.', '/');
+            }
+            else if (brokerageSymbol.Contains('-'))
+            {
+                brokerageSymbol.Replace('-', 'p');
+            }
+            else if(brokerageSymbol.Contains('+'))
+            {
+                brokerageSymbol.Replace("+", "/WS");
+            }
```

Comment:
No need to call `Contains` can directly call `Replace`?

Reply:
I changed to one line with helping 🙄 thanks